ceph.md

```diff
@@ -55,6 +55,9 @@ cephadm shell -- \
 	 mon 'allow r' \
 	 osd 'allow class-read object_prefix rbd_children, allow rwx pool=vms, allow rwx pool=volumes, allow rwx pool=images'
 ```
+If Manila is enabled in the OpenStack controlplane, then add `allow
```

Comment:
+1 given we already included the `gs volume create` on L47

Reply:
Also, I'm wondering if in L54 would be enough having `mgr 'allow rw'` instead of `*`. In my env (which is the same key used in NG CI) we have [1] and we could think about updating L54 accordingly.
However, I see in TripleO we have [2], so I'm not sure which service requires `allow *` instead of `rw`.
@fultonj wdyt?

[1] https://github.com/openstack-k8s-operators/install_yamls/blob/main/scripts/gen-ceph-kustomize.sh#L177 
[2] https://github.com/openstack/tripleo-ansible/blob/master/tripleo_ansible/roles/tripleo_run_cephadm/tasks/build_keys.yml#L32

Reply:
I'll update it to `rw` instead of `*`. thanks